package/tests/conftest.py

```diff
@@ -145,151 +140,53 @@ def example_api(
-        example_pipelines,
-        example_session_store_location,
-    )
-    with mock.patch(
-        "kedro_viz.api.responses.data_access_manager", new=data_access_manager
-    ), mock.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager):
-        yield api
+    populate_data(data_access_manager, example_catalog, example_pipelines, None)
+    mocker.patch("kedro_viz.api.responses.data_access_manager", new=data_access_manager)
+    mocker.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager)
+    yield api
 
 
 @pytest.fixture
 def example_api_no_session_store(
     data_access_manager: DataAccessManager,
     example_pipelines: Dict[str, Pipeline],
     example_catalog: DataCatalog,
+    mocker,
 ):
     api = apps.create_api_app_from_project(mock.MagicMock())
     populate_data(data_access_manager, example_catalog, example_pipelines, None)
-    with mock.patch(
-        "kedro_viz.api.responses.data_access_manager", new=data_access_manager
-    ), mock.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager):
-        yield api
+    mocker.patch("kedro_viz.api.responses.data_access_manager", new=data_access_manager)
+    mocker.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager)
+    yield api
 
 
 @pytest.fixture
 def example_transcoded_api(
     data_access_manager: DataAccessManager,
     example_transcoded_pipelines: Dict[str, Pipeline],
     example_transcoded_catalog: DataCatalog,
-    example_session_store_location: Optional[Path],
+    mocker,
 ):
     api = apps.create_api_app_from_project(mock.MagicMock())
     populate_data(
         data_access_manager,
         example_transcoded_catalog,
         example_transcoded_pipelines,
-        example_session_store_location,
+        None,
     )
-    with mock.patch(
-        "kedro_viz.api.responses.data_access_manager", new=data_access_manager
-    ), mock.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager):
-        yield api
+    mocker.patch("kedro_viz.api.responses.data_access_manager", new=data_access_manager)
+    mocker.patch("kedro_viz.api.router.data_access_manager", new=data_access_manager)
+    yield api
 
 
 @pytest.fixture
 def client(example_api):
     yield TestClient(example_api)
 
 
-@pytest.fixture
-def example_session_store_location(tmp_path):
-    yield Path(tmp_path / "session_store.db")
-
-
-@pytest.fixture
-def example_db_session(example_session_store_location):
-    engine = create_engine(f"sqlite:///{example_session_store_location}")
-    Base.metadata.create_all(engine)
-    Session = sessionmaker(bind=engine)
-    session = Session()
-    yield session
-    session.close()
-
-
-@pytest.fixture
-def save_version():
-    yield "2021-11-02T18.24.24.379Z"
-
-
-@pytest.fixture
-def save_new_version():
-    yield "2021-11-03T18.24.24.379Z"
-
-
-@pytest.fixture
-def example_db_dataset(example_db_session, save_version, save_new_version):
```

Comment:
I moved this whole thing to a conftest dedicated to graphql test. Also refactored it a little bit.